src/api/quoter/quote/quote.ts

```diff
@@ -104,6 +106,18 @@ export class Quote {
         return this.presets[type]
     }
 
+    private handlePredicate(params: PredicateParams): string {
+        if (params?.nonce && params.nonce !== OrderNonce.Empty) {
```

Comment:
`params?.nonce` this check will be failed in case of `nonce === 0`. Check for `undefined` only, since I've asked to remove `OrderNonce` from `PredicateParams.nonce`